### PR TITLE
fix temporary password regex

### PR DESCRIPTION
**Pull request by yokha**

### CODE REVIEW

File: users/create_user.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func getTemporaryPassword(emailMessage string) (string, error) {
 	return subMatches[0][1], nil
 }
 
-var temporaryPasswordRegexp = regexp.MustCompile(`password=(\S+)\" style`)
+var temporaryPasswordRegexp = regexp.MustCompile(`password=(\S+)\"\s+\S+`)
```
Comment (Ekberg) [inline users/create_user.go:101]: Something like this should be more robust and future proof,
```suggestion
// Matching links like https://digital-services.skf.com/sign-in?user_name=<username>&password=<temporary_password>
var temporaryPasswordRegexp = regexp.MustCompile(`\?(?:[^&]+&)*password=([^\"&]+)(?:\"|&)`)
```

Comment (Ekberg) [reply users/create_user.go:101]: Some unit tests might be nice to add

**Review by Ekberg — CHANGES_REQUESTED** [review]

File: users/create_user.go (hunk 1 above)
Comment (yokha) [reply users/create_user.go]: I changed regex as you suggested with aslight change to error if password is starting with space. I added some unitest.

Comment (yokha) [reply users/create_user.go]: No need for password rule checks. reverted change on the regex.

**Review by Ekberg — APPROVED** [review]

**Pull request merged**